frontend/src/views/WheresMyGeneV2/components/GeneSearchBar/components/ShareButton/index.tsx

```diff
@@ -40,8 +78,13 @@ export default function ShareButton(): JSX.Element {
       filters: selectedFilters,
       organism: selectedOrganismId,
       genes: selectedGenes,
+      cellTypes: filteredCellTypes,
     });
 
+    const filteredCellTypeIDs = filteredCellTypes.map(
```

Comment:
To keep the code DRY, maybe worth extracting a function to get `filteredCellTypeIDs` ? I see similar logic here and below 🙏 Thank you!